### DISCUSSION TITLE
is opera supported?

### DISCUSSION THREAD

**Discussion by Baloo9753** [aclap-dev/video-downloadhelper#384] [Help and Q&A]
### Your question

Hello, all I see ist Firefox and Chrome ... but Opera is not explicit written ...
because I buyed a license for Firefox (it worked fine up to today (see onlsfans not loadable) I wonder if Opera is also supported - will it need an additional license?

THANK YOU

### Addon & CoApp version and details

Win 11 Pro, latest release 
Firefox latest
Opera latest
very fast intenet connection

**Comment by jcfrog** [comment]
Hi, sorry we only support Chrome, Edge and Firefox for the moment.

**Comment by Cronosus** [comment]
opera have chrome core, so it works with chrome plugins (and need ofc chrome licence)
